components/scream/src/physics/p3/p3_functions_f90.cpp

```diff
@@ -994,6 +1003,56 @@ void get_rain_dsd2_f(Real qr_, Real* nr_, Real* mu_r_, Real* lamr_, Real* cdistr
   *logn0r_ = t_h(4);
 }
 
+void get_time_space_phys_variables_f(Real t_, Real pres_, Real rho_, Real xxlv_, Real xxls_, Real qvs_, Real qvi_,
+				     Real* mu_, Real* dv_, Real* sc_, Real* dqsdt_, Real* dqsidt_, Real* ab_,
+				     Real* abi_, Real* kap_, Real* eii_)
+{
+  using P3F = Functions<Real, DefaultDevice>;
+
+  typename P3F::view_1d<Real> t_d("t_h", 9);
+  auto t_h = Kokkos::create_mirror_view(t_d);
+
+  Real local_mu     = *mu_;
```

Comment:
I don't think any of these local_* are needed.

Reply:
I am a little confused here. These are intent outs. Don't we need them in the following:
```
typename P3F::Spack mu(local_mu), dv(local_dv), sc(local_sc), dqsdt(local_dqsdt),
        dqsidt(local_dqsidt), ab(local_ab), abi(local_abi), kap(local_kap), eii(local_eii);
```

Reply:
All these values are intent(out), so there's no need to get there values as inputs. You can initialize the corresponding Spacks with their default constructor.

Reply:
Thanks! I have fixed it now.